bin/BuildPackages.sh

```diff
@@ -60,36 +30,52 @@ then
         "Type: cd ../pkg; ../bin/BuildPackages.sh"
 fi
 
-# CURDIR
 CURDIR="$(pwd)"
+GAPDIR="$(cd .. && pwd)"
+COLORS=yes
+PACKAGES=
+
+# If output does not go into a terminal (but rather into a log file),
+# turn of colors.
```

Comment:
I was also wondering about the style of the comments. It would be nice to unify them. Currently some comments are complete sentences, some are only words, some are half sentences starting with small letters, etc. This of course need not be part of this PR.